packages/react/src/QuestionnaireForm/QuestionnaireForm.tsx

```diff
@@ -188,7 +190,7 @@ function QuestionnaireFormItemArray(props: QuestionnaireFormItemArrayProps): JSX
 
   if (props.renderPages) {
     return (
-      <Stepper active={props.activePage ?? 0} allowNextStepsSelect={false}>
+      <Stepper active={props.activePage ?? 0} allowNextStepsSelect={false} style={{ padding: 6 }}>
```

Comment:
Stepper props: https://mantine.dev/core/stepper/?t=props

Can you try `p={6}`?